addon/-private/adapters/build-url-mixin.js

```diff
@@ -434,7 +434,7 @@ export default Ember.Mixin.create({
     @return {String} path
   **/
   pathForType(modelName) {
-    var camelized = Ember.String.camelize(modelName);
+    const camelized = Ember.String.camelize(modelName);
```

Comment:
yes, I am going back through these PRs per @bmac's suggestion:
"I believe Ember follows a convention of "liberally-let" per madhatted.com/2016/1/25/let-it-be and only allows const in module scope."

I'll update once the builds are working again.